Hymns/Favorites/FavoritesView.swift

```diff
@@ -4,30 +4,30 @@ import SwiftUI
 
 struct FavoritesView: View {
 
-    @ObservedObject private var viewModel: FavoritesViewModel
+    @ObservedObject private var viewModel: TagsViewModel
```

Comment:
This shouldn't be renamed. A view model should be associated with the view that it's supporting, so just because under the hood, Favorites the same thing as tags, doesn't mean we should expose that to the view. That should be abstracted away from the view.

Reply:
Addressed in 783aae0